code/modules/clothing/suits/marine_armor.dm

```diff
@@ -342,72 +337,74 @@ var/list/squad_colors_chat = list(rgb(230,125,125), rgb(255,230,80), rgb(255,150
 
 /obj/item/clothing/suit/storage/marine/MP/warden
 	icon_state = "warden"
-	name = "\improper M3 pattern warden MP armor"
-	desc = "A well-crafted suit of M3 Pattern Armor typically distributed to Wardens. Useful for letting your men know who is in charge."
+	name = "\improper M2 pattern warden MP armor"
+	desc = "A well-crafted suit of M2 Pattern Armor typically distributed to Wardens. Useful for letting your men know who is in charge."
 	armor_bio = CLOTHING_ARMOR_MEDIUMLOW
 	armor_rad = CLOTHING_ARMOR_MEDIUMLOW
+	flags_armor_protection = BODY_FLAG_CHEST|BODY_FLAG_GROIN|BODY_FLAG_ARMS|BODY_FLAG_LEGS
 	uniform_restricted = list(/obj/item/clothing/under/marine/warden)
-	specialty = "M3 pattern warden MP"
+	specialty = "M2 pattern warden MP"
 	item_state_slots = list(WEAR_JACKET = "warden")
 
 /obj/item/clothing/suit/storage/marine/MP/warden/padless
-	name = "\improper M3 pattern padless warden MP armor"
+	name = "\improper M2 pattern padless warden MP armor"
 	icon_state = "warden_2"
-	specialty = "M3 pattern padless warden MP"
+	specialty = "M2 pattern padless warden MP"
 	item_state_slots = list(WEAR_JACKET = "warden_2")
 
 /obj/item/clothing/suit/storage/marine/MP/warden/jacket
-	name = "\improper warden service jacket over M3 pattern warden MP armor"
-	desc = "A well-crafted suit of M3 Pattern Armor typically distributed to Wardens, with a Warden service jacket worn on top. Look your best while you're escorting prisoners to their execution."
+	name = "\improper warden service jacket over M2 pattern warden MP armor"
+	desc = "A well-crafted suit of M2 Pattern Armor typically distributed to Wardens, with a Warden service jacket worn on top. Look your best while you're escorting prisoners to their execution."
 	icon_state = "warden_jacket"
-	specialty = "warden service jacket over M3 pattern warden MP"
+	specialty = "warden service jacket over M2 pattern warden MP"
 	item_state_slots = list(WEAR_JACKET = "warden_jacket")
 
 /obj/item/clothing/suit/storage/marine/MP/WO
 	icon_state = "warrant_officer"
-	name = "\improper M3 pattern chief MP armor"
-	desc = "A well-crafted suit of M3 Pattern Armor typically distributed to Chief MPs. Useful for letting your men know who is in charge."
+	name = "\improper M2 pattern chief MP armor"
+	desc = "A well-crafted suit of M2 Pattern Armor typically distributed to Chief MPs. Useful for letting your men know who is in charge."
 	uniform_restricted = list(/obj/item/clothing/under/marine/officer/warrant)
-	specialty = "M3 pattern chief MP"
+	flags_armor_protection = BODY_FLAG_CHEST|BODY_FLAG_GROIN|BODY_FLAG_ARMS|BODY_FLAG_LEGS
```

Comment:
same thing here, the stats aren't different normally so why is it stronger

Reply:
its a special issued M2 as it says on the description

Reply:
also its a different sprite